content/docs/distribution/customer_returns.md

```diff
@@ -393,7 +394,7 @@ Once you have done this:
 - Return status is now `DELIVERED`
 
 <div class="note">
-In case your Customer Return is <b>automatic</b>, you cannot confirm its delivery unless the customer has confirmed its shipment. In other words, your shipment status has to be <code>SHIPPED</code> before you can confirm that you have received the returned goods. 
+In case your Customer Return is <b>automatic</b>, you cannot confirm its delivery unless the customer has confirmed its shipment. In other words, your invoice status has to be <code>SHIPPED</code> before you can confirm that you have received the returned goods. 
```

Comment:
I thought we were quite conscious _not_ to use the word invoice? I get it in places where you are talking about different kinds of invoices, but I think this should just say `return status` no?